holoviews/plotting/mpl/plot.py

```diff
@@ -262,9 +262,6 @@ class GridPlot(CompositePlot):
     show_legend = param.Boolean(default=False, doc="""
         Legends add to much clutter in a grid and are disabled by default.""")
 
-    tick_format = param.String(default="%.2f", doc="""
```

Comment:
It will at least need to be mentioned in the CHANGELOG. Other than that I think it is fine.